<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Tester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0">
     <widget class="QLabel" name="inputLabel">
      <property name="text">
       <string>Input</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="effectiveInputLabel">
      <property name="text">
       <string>Effective Input</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="emaOutputLabel">
      <property name="text">
       <string>EMA Filter Output</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="emaFilterSettings">
        <property name="title">
         <string>EMA Filter Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="emaFilterLowerBoundLabel">
           <property name="text">
            <string>Lower Bound</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="emaFilterLowerBoundSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="emaFilterUpperBoundLabel">
           <property name="text">
            <string>Upper Bound</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="emaFilterUpperBoundSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="emaFilterSnapMultiplierLabel">
           <property name="text">
            <string>Snap Multiplier</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="emaFilterSnapMultiplierSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="enableSnaptoEdgesCheckBox">
           <property name="text">
            <string>Allow Snapping to edges</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="enableSleepCheckBox">
           <property name="text">
            <string>Allow Sleeping</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="emaActivityThresholdLabel">
           <property name="text">
            <string>Activity Threshold</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="emaActivityThresholdSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="simpleFilterSettings">
        <property name="title">
         <string>Simple Filter Settings</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="simpleFilterSuppressionCountLabel">
           <property name="text">
            <string>Suppression Count</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="simpleFilterSuppressionCountSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="simpleFilterActivityThresholdLabel">
           <property name="text">
            <string>Activity Threshold</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="simpleFilterThresholdSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="noiseSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="enableNoiseCheckBox">
      <property name="text">
       <string>Noise</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="simpleFilterOutputLabel">
      <property name="text">
       <string>Simple Filter Output</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="enableFiltersCheckbox">
      <property name="text">
       <string>Filters Enabled</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QSlider" name="emaOutputSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSlider" name="effectiveInputSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSlider" name="inputSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QSlider" name="simpleOutputSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QSpinBox" name="effectiveInputSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QSpinBox" name="simpleOutputSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QSpinBox" name="emaOutputSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="inputSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
